rewrite-core/src/main/java/org/openrewrite/text/CreateTextFile.java

```diff
@@ -86,7 +86,7 @@ public TreeVisitor<?, ExecutionContext> getVisitor(AtomicBoolean created) {
             @Override
             public SourceFile visit(@Nullable Tree tree, ExecutionContext ctx) {
                 SourceFile sourceFile = (SourceFile) requireNonNull(tree);
-                if ((created.get() || Boolean.TRUE.equals(overwriteExisting)) && path.equals(sourceFile.getSourcePath())) {
```

Comment:
Weird how we were overwriting the content again, even after already initializing with file contents.